### PR TITLE
add SPEEDYBEEF7MINIV2

### PR DESCRIPTION
**Pull request by SEA729**
Add SPEEDYBEEF7MINIV2 targets and CODEOWNERS
thanks for your review

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -104,3 +104,4 @@
 /configs/default/WIZZ-* @bnn1044 @betaflight/unified-target-administrators
 /configs/default/YUPF-* @Faduf @betaflight/unified-target-administrators
 /configs/default/ZEEZ-* @Guidus93 @betaflight/unified-target-administrators
+/configs/default/SPBE-* @SEA729 @betaflight/unified-target-administrators
```
Comment (haslinghuis) [inline .github/CODEOWNERS:107]: Remove as `CODEOWNERS` file is maintained by Betaflight.

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: configs/default/SPBE-SPEEDYBEEF7MINIV2.config [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+# pin C08: DMA2 Stream 2 Channel 0
+dma pin A10 0
+# pin A10: DMA2 Stream 6 Channel 0
+dma pin A01 0
+# pin A01: DMA1 Stream 4 Channel 6
+dma pin A00 0
+# pin A00: DMA1 Stream 2 Channel 6
+
+# feature
+feature -RX_PARALLEL_PWM
```
Comment (haslinghuis) [inline configs/default/SPBE-SPEEDYBEEF7MINIV2.config:111]: Please remove as setting RX_SERIAL will disable the feature

File: .github/CODEOWNERS (hunk 1 above)
Comment (SEA729) [reply .github/CODEOWNERS]: thanks for your review

File: configs/default/SPBE-SPEEDYBEEF7MINIV2.config (hunk 2 above)
Comment (SEA729) [reply configs/default/SPBE-SPEEDYBEEF7MINIV2.config]: thanks for your review

**Review by haslinghuis — APPROVED** [review]

**Review by Asizon — APPROVED** [review]

**Pull request merged**